BookPlayer/Settings/SettingsViewController.swift

```diff
@@ -7,58 +7,119 @@
     }
 
     @objc func globalSpeedToggleDidChange() {
-        defaults.set(globalSpeedSwitch.isOn, forKey: UserDefaultsConstants.globalSpeedEnabled)
+        UserDefaults.standard.set(self.globalSpeedSwitch.isOn, forKey: UserDefaultsConstants.globalSpeedEnabled)
     }
 
     @objc func autoplayToggleDidChange() {
-        defaults.set(autoplaySwitch.isOn, forKey: UserDefaultsConstants.autoplayEnabled)
+        UserDefaults.standard.set(self.autoplaySwitch.isOn, forKey: UserDefaultsConstants.autoplayEnabled)
     }
 
     @IBAction func done(_ sender: UIBarButtonItem) {
         self.dismiss(animated: true, completion: nil)
     }
 
-    //TODO: remove this once settings page is completed
-//    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-//        //hide all options except smart rewind
-//        switch section {
-//        case 0, 1:
-//            return 0
-//        default:
-//            return 4
-//        }
-//    }
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath as IndexPath, animated: true)
+
+        switch indexPath {
+            case self.supportEmailPath:
+                self.sendSupportEmmail()
+            case self.githubLinkPath:
+                self.showProjectOnGitHub()
+            default: break
+        }
+    }
+
+    func mailComposeController(_ controller: MFMailComposeViewController, didFinishWith result: MFMailComposeResult, error: Error?) {
+        controller.dismiss(animated: true)
+    }
+
+    func sendSupportEmmail() {
+        let device = Device()
+        let appVersion = "\(self.version)-\(self.build)"
+        let systemVersion = "\(UIDevice.current.systemName) \(UIDevice.current.systemVersion)"
+
+        if MFMailComposeViewController.canSendMail() {
+            let mail = MFMailComposeViewController()
+
+            mail.mailComposeDelegate = self
+            mail.setToRecipients([self.supportEmail])
+            mail.setSubject("I need help with BookPlayer \(self.version)-\(self.build)")
+            mail.setMessageBody("<p>Hello BookPlayer Crew,<br>I have an issue concerning BookPlayer \(appVersion) on my \(device) running \(systemVersion)</p><p>When I try to…</p>", isHTML: true)
+
+            self.present(mail, animated: true)
+        } else {
+            let debugInfo = "BookPlayer \(appVersion)\n\(device) with \(systemVersion)"
+
+            let alert = UIAlertController(title: "Unable to compose email", message: "You need to set up an email account in your device settings to use this. \n\nPlease mail us at \(self.supportEmail)\n\n\(debugInfo)", preferredStyle: .alert)
+
+            alert.addAction(UIAlertAction(title: "Copy information to clipboard", style: .default, handler: { _ in
+                UIPasteboard.general.string = "\(self.supportEmail)\(debugInfo)"
```

Comment:
This needs a `\n` between the `supportEmail` and `debugInfo` otherwise when pasted, the string would be `support@bookplayer.appBookPlayer 3.0.0...`